### PR TITLE
Update objects and database to support time due

### LINKED ISSUE #121: Need Time due to arcs and tasks
Currently there is no way for users to see what time a task or arc is due by, they can only see the date. In order to add time we can do two things:

1) Make the `dueDate` in arc and task also contain the time
2) Add another time attribute to both arc and task

I believe it is best to go with option 2 as many of our date related functions compare date strings and adding time to the end of the strings may make functions more complicated then needed.

### PR DESCRIPTION
**Pull request by KevinKelly25**
This PR adds time due to tasks and arcs objects. This PR does not add this to the UI but sets the objects up so it can be easily stored and read from the objects with the current functions within the app.

This PR does not update the PostgreSQL tables with this attribute as it would interrupt the currently outstanding PR #84 

Closes #121

### CODE REVIEW

**Review by jgrabowski121 — APPROVED** [review]
Thank you @KevinKelly25 , this should make the addition of a Time field to the UI easier

File: client/src/arcplanner/lib/src/util/databaseHelper.dart [hunk 1]
```diff
@@ -116,13 +119,14 @@ class DatabaseHelper {
...
-          $_taskDueDate TEXT, 
+          $_taskDueDate TEXT,
+          $_taskTimeDue TEXT,
           $_taskLoc TEXT CHECK(LENGTH($_taskLoc) <= $_locSize),
           $_taskCompleted INTEGER CHECK($_taskCompleted == 0 OR $_taskCompleted == 1)
           )""");
     await db.execute("""
         CREATE VIEW $_arcView AS
-        SELECT UID, AID, Title, Description, DueDate, ParentArc, Completed, 
+        SELECT UID, AID, Title, Description, DueDate, TimeDue ParentArc, Completed,
```
Comment (jrm86) [inline client/src/arcplanner/lib/src/util/databaseHelper.dart]: Should be a comma between TimeDue and ParentArc. Does not affect the operation of the DB, but for consistency.

**Comment by jrm86** [comment]
Good to go. There's a comma missing on line 129 of databaseHelper.dart between TimeDue and ParentArc. Doesn't affect the behavior of the app - just a consistency thing.

**Review by jrm86 — APPROVED** [review]

**Comment by KevinKelly25** [comment]
Fixed that error @jrm86 . Thank you for seeing that error.

**Review by MatthewChastain — APPROVED** [review]
Looks good @KevinKelly25, thank you for this addition!

**Pull request merged**